tests/test_cli.py

```diff
@@ -81,7 +81,7 @@ def test_no_traces(store_data, stdout, stderr):
     store, db_file = store_data
     with mock.patch.dict(os.environ, {DefaultConfig.DB_PATH_VAR: db_file.name}):
         ret = cli.main(['stub', func.__module__], stdout, stderr)
-    assert stderr.getvalue() == "No traces found\n"
+    assert stderr.getvalue() == f"No traces found for module path ('{func.__module__}', None)\n"
```

Comment:
I think this `(module, qualname)` representation will be confusing, especially when qualname is `None` (which is the more common case). Can we rejoin the tuple into its original form instead? (Or maybe there is some argparse way to preserve the original pre-parsing string?)